vignettes/markdown_notation.Rmd

```diff
@@ -24,7 +24,7 @@ knitr::opts_chunk$set(
 params$region <- "NWFSC"
 ```
 
-## Markdown Notation
+# Markdown Notation
```

Comment:
I was taught that if a subordinate header is used (e..g, a second-level header), then a first-level header should be present before the second one. This is a good practice for both UX and accessibility, since those using screen readers will assume the material is organized with top-level headers first, then second, etc. Would you prefer that I revert the header changes and, for example, add a new top-level header? Or totally revert my changes?

Reply:
Hmm yeah if that's the better practice then better to keep it the way you set and I will make the adjustments to the other headers accordingly